compat/vcbuild/scripts/clink.pl

```diff
@@ -12,10 +12,11 @@
 use strict;
 my @args = ();
 my @cflags = ();
+my @lflags = ();
 my $is_linking = 0;
 while (@ARGV) {
 	my $arg = shift @ARGV;
-	if ("$arg" =~ /^-[DIMGO]/) {
+	if ("$arg" =~ /^-[DIMGOZ]/) {
```

Comment:
What is `-Z` supposed to do? Could you describe that in the commit message, and if necessary, add an argument why we simply pass it through?